src/networkremote/remotesettings.h

```diff
@@ -0,0 +1,32 @@
+#ifndef REMOTESETTINGS_H
+#define REMOTESETTINGS_H
+
+#include <QObject>
+#include "core/settings.h"
+
+class NetworkRemoteSettings
+{
+public:
+  static const char *kSettingsGroup;
+  explicit NetworkRemoteSettings();
+  ~NetworkRemoteSettings();
+  void Load();
+  void Save();
+  bool UserRemote();
+  bool LocalOnly();
+  QString GetIpAddress();
+  int GetPort();
+  void SetUseRemote(bool);
+  void SetLocalOnly(bool);
+  void SetIpAdress ();
+  void SetPort(int);
+
+private:
+  QSettings s_;
```

Comment:
QSettings doesn't need to be a member variable

Reply:
Not sure what you mean